autokeras/auto/processor.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+
+
+class OneHotEncoder:
```

Comment:
`(object)`

Reply:
Keep in mind that this whole file will end up deprecated with preprocessing layers and tf.data. Don't put too much work into it.